external/libp2p-service.js

```diff
@@ -193,8 +193,9 @@ class Libp2pService {
                     }
                 }
             } catch (e) {
+                const stringifiedData = await this.workerPool.exec('JSONStringify', [data]);
```

Comment:
This is compute intensive operation, let's consider if we want to exclude it in the next release